### PR TITLE
IMS

### PR DESCRIPTION
**Pull request by elektroschmock**

### CODE REVIEW

File: shamu/Android.mk [hunk 1]
```diff
@@ -19,203 +19,4 @@ LOCAL_PATH := $(call my-dir)
...
-include $(CLEAR_VARS)
-LOCAL_MODULE := libtime_genoff
-LOCAL_MODULE_OWNER := motorola
-LOCAL_SRC_FILES := proprietary/vendor/lib/libtime_genoff.so
-LOCAL_MULTILIB := 32
-LOCAL_MODULE_TAGS := optional
-LOCAL_MODULE_CLASS := SHARED_LIBRARIES
-LOCAL_MODULE_SUFFIX := .so
-LOCAL_VENDOR_MODULE := true
-include $(BUILD_PREBUILT)
```
Comment (ghost) [inline shamu/Android.mk:31]: Looks like a horrible idea to me

Comment (npjohnson) [reply shamu/Android.mk:31]: You are aware this is migrated to Android.bp as of this change, yes?

Comment (ghost) [reply shamu/Android.mk:31]: > You are aware this is migrated to Android.bp as of this change, yes?

I'm not. Thanks for bringing that up. I really should look at the changes appropriately before commenting on them. Again thank you.

**Comment by npjohnson** [comment]
I was never able to get this to work :/

**Pull request closed without merging**